env/staging/ecs/task-definitions/metrics.json

```diff
@@ -73,7 +73,7 @@
 		],
 		"cpu": 10,
 		"essential": true,
-		"image": "falcosecurity/falco:0.17.1",
+		"image": "falcosecurity/falco:master",
```

Comment:
Instead of pinning to what I'm assuming is a mutable tag is it possible to pin to a specific version?

Reply:
Pinning to master is the recommended approach i'm proposing since it allows us to stay on top of kernel header updates since we're in a serverless configuration where the underlying linux AMI can change versions at any time.

There's a lower risk of a bad container from using master vs an interruption of falco services due to missing kernel drivers.